app/views/report_mailer/tables_report.html.erb

```diff
@@ -2,6 +2,16 @@
 - @message: text to put in the beginning of the email
 - @reports: an array of EmailableReport
 %>
+<header class="usa-header">
+  <%= image_tag(
+    asset_url('logo.svg'),
```

Comment:
I think the build should catch it, but SVG images are not well-supported in email.

https://www.caniemail.com/features/image-svg/

There's a PNG version you can use in https://github.com/18F/identity-idp/tree/main/app/assets/images/email

Reply:
Ah, actually, the build may not catch it, since we're only testing `UserMailer`.

Reply:
> Ah, actually, the build may not catch it, since [we're only testing `UserMailer`](https://github.com/18F/identity-idp/blob/8bd41a8c835b57ab32cd79aaf5063fa9a2606b1b/spec/mailers/previews/user_mailer_preview_spec.rb#L13-L18).

To be improved in #11446